libs/traits/src/benchmarking.rs

```diff
@@ -67,6 +67,7 @@ pub trait ForeignInvestmentBenchmarkHelper {
 	type Balance;
 	type CurrencyId;
 	type InvestmentId;
+	type SetupInfo;
```

Comment:
I think it doesn't need to be generalized here. The new struct can just be used directly and even can be in the traits module (i.e. just before the trait). I mean, it's not a type the user needs to configure in runtime or similar. It's just an aggregation of fields to avoid using a nameless tuple. WDYT?

Reply:
That was my initial thought but then I assumed you'd prefer it to be generic. Will follow your proposal then to make it absolutely explicit!